browser/ui/webui/playlist_ui.cc

```diff
@@ -30,9 +36,31 @@ bool PlaylistUI::ShouldBlockPlaylistWebUI(
 }
 
 PlaylistUI::PlaylistUI(content::WebUI* web_ui, const std::string& name)
-    : MojoWebUIController(web_ui) {
-  CreateAndAddWebUIDataSource(web_ui, name, kPlaylistGenerated,
-                              kPlaylistGeneratedSize, IDR_PLAYLIST_HTML);
+    : UntrustedWebUIController(web_ui) {
+  // From MojoWebUIController
+  web_ui->SetBindings(content::BINDINGS_POLICY_MOJO_WEB_UI);
+
+  auto* source =
+      CreateAndAddWebUIDataSource(web_ui, name, kPlaylistGenerated,
+                                  kPlaylistGeneratedSize, IDR_PLAYLIST_HTML);
+
+  // Allow to load untrusted resources.
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::StyleSrc,
+      std::string("style-src chrome-untrusted://resources "
+                  "chrome-untrusted://brave-resources 'unsafe-inline';"));
```

Comment:
does this really need inline styles?

Reply:
Yeah, I guess so because of styled-component we're using. I'm far from FE expert, but from my short research, I guess we can't use `nonce` either because our nonce would be fixed at compile time. And many Web UIs seem to be still using 'unsafe-inline' for style-src.

@petemill @fallaciousreasoning Could you tell me anything about this?

Reply:
np, seems ok then